src/constants.ts

```diff
@@ -196,16 +196,16 @@ export const APP_PATH: { [routeId in RouteId]: RouteInfo } = {
 		showInContentPicker: true,
 		showForInteractiveTour: true,
 	},
-	ASSIGNMENT_CREATE: {
-		route: `/${ROUTE_PARTS.workspace}/${ROUTE_PARTS.assignments}/${ROUTE_PARTS.create}`,
-		showInContentPicker: true,
-		showForInteractiveTour: true,
-	},
 	ASSIGNMENT_DETAIL: {
```

Comment:
```suggestion
	// This entry must come before the ASSIGNMENT_CREATE route, otherwise the interactive tour button doesn't correctly identify the route
	// https://meemoo.atlassian.net/browse/AVO-2923
	ASSIGNMENT_DETAIL: {
```